rcsb/exdb/wf/PubChemEtlWorkflow.py

```diff
@@ -47,23 +47,48 @@ def __init__(self, **kwargs):
             logger.setLevel(logging.DEBUG)
         #
 
-    def dump(self):
-        """Dump the current object store of PubChem correspondences and data."""
+    def dump(self, **kwargs):
+        """Dump the current object store of PubChem correspondences and data.
+
+        Args:
+            useStash(bool):  should stash (Buildlocker) be updated? (default: True)
+            useGit(bool):  should stash (GitHub) be updated? (default: True)
+
+        Returns:
+            (bool): True for success or False otherwise
+
+        """
         ok1 = ok2 = ok3 = ok4 = False
         try:
+            useStash = kwargs.get("useStash", True)
+            useGit = kwargs.get("useGit", True)
```

Comment:
So, although you're turning off the git stashing from occurring during the weekly Sunday Azure tests, does this mean that when we start running this weekly on Wednesdays, it'll start pushing to Git again weekly? If so, I guess we'll want to turn that off as well at some point.

Reply:
That applies to all the stashed files, though. We still need to decide if we want to continue to have GitHub as the backup stash and fall-back.